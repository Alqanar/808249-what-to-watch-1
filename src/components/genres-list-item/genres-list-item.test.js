import React from "react";
import renderer from "react-test-renderer";

import GenresListItem from "./genres-list-item.jsx";


it(`Genres list item correctly renders`, () => {
  const tree = renderer
    .create(<GenresListItem
      name="Dramas"
      onGenreClick={() => {}}
      isActive={false}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
