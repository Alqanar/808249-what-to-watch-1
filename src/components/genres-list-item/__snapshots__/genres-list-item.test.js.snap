// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Genres list item correctly renders 1`] = `
<li
  className="catalog__genres-item "
>
  <a
    className="catalog__genres-link"
    href="#"
    onClick={[Function]}
  >
    Dramas
  </a>
</li>
`;
